test/vmtester/tests.cpp

```diff
@@ -110,7 +110,7 @@ TEST_F(evmc_vm_test, execute_create)
     }
     else
     {
-        EXPECT_NE(result.output_size, 0);
+        EXPECT_NE(result.output_size, size_t{0});
```

Comment:
Why are these needed?

Reply:
A warning in MSVC about comparing signed and unsigned types.